### PR TITLE
CLOUDP-200398: docs corrections

### PR DESCRIPTION
**Pull request by tibulca**
## Proposed changes
Docs updates based on https://github.com/mongodb/mongodb-atlas-cli/pull/2263

_Jira ticket:_ CLOUDP-200398

### CODE REVIEW

File: docs/atlascli/command/atlas-clusters-create.txt [hunk 1]
```diff
@@ -115,7 +115,7 @@ Options
    * - -r, --region
      - string
      - false
-     - Physical location of your MongoDB cluster. You must specify the region option if you don't use the --file option. For a complete list of supported AWS regions, see: https://www.mongodb.com/docs/atlas/reference/amazon-aws/. For a complete list of supported Azure regions, see: https://www.mongodb.com/docs/atlas/reference/microsoft-azure/#microsoft-azure. For a complete list of supported GCP regions, see: https://www.mongodb.com/docs/atlas/reference/google-gcp/.
+     - Physical location of your MongoDB cluster. You must specify the region option if you don't use the --file option. For a complete list of supported AWS regions, see: https://dochub.mongodb.org/core/aws-atlas. For a complete list of supported Azure regions, see: https://dochub.mongodb.org/core/azure-atlas. For a complete list of supported GCP regions, see: https://dochub.mongodb.org/core/gcp-atlas.
```
Comment (tibulca) [inline docs/atlascli/command/atlas-clusters-create.txt:118]: using persistent URLs, see https://github.com/mongodb/mongodb-atlas-cli/pull/2263#discussion_r1330078839

File: docs/atlascli/command/atlas-setup.txt [hunk 2]
```diff
@@ -88,15 +88,15 @@ Options
...
+     - Physical location of your MongoDB cluster. For a complete list of supported AWS regions, see: https://dochub.mongodb.org/core/aws-atlas. For a complete list of supported Azure regions, see: https://dochub.mongodb.org/core/azure-atlas. For a complete list of supported GCP regions, see: https://dochub.mongodb.org/core/gcp-atlas..
    * - --skipMongosh
      - 
      - false
      - Indicates whether to skip accessing your deployment with MongoDB Shell.
    * - --skipSampleData
      - 
      - false
-     - Indicates whether to skip loading sample data into your Atlas cluster.
+     - Indicates whether to skip loading sample data into your MongoDB cluster.
```
Comment (tibulca) [inline docs/atlascli/command/atlas-setup.txt:99]: see https://github.com/mongodb/mongodb-atlas-cli/pull/2263#discussion_r1330081177

**Review by blva — DISMISSED** [review]

File: docs/atlascli/command/atlas-setup.txt [hunk 3]
```diff
@@ -40,7 +40,7 @@ Options
    * - --accessListIp
      - strings
      - false
-     - IP address to be allowed to access the deployment.
+     - IP address to grant access to the deployment.
```
Comment (tibulca) [inline docs/atlascli/command/atlas-setup.txt:43]: see https://github.com/mongodb/mongodb-atlas-cli/pull/2263#discussion_r1330060782

File: internal/cli/atlas/deployments/setup.go [hunk 4]
```diff
@@ -799,7 +799,7 @@ func SetupBuilder() *cobra.Command {
 		Short: "Create a local deployment.",
 		Args:  require.MaximumNArgs(1),
 		Annotations: map[string]string{
-			"deploymentNameDesc": "Name of the deployment that you want to setup.",
+			"deploymentNameDesc": "Name of the deployment that you want to set up.",
```
Comment (tibulca) [inline internal/cli/atlas/deployments/setup.go:802]: see https://github.com/mongodb/mongodb-atlas-cli/pull/2263#discussion_r1330060371

**Pull request merged**